src/Model/Command/UserRecommendation.php

```diff
@@ -123,6 +125,33 @@ public function setFilters(array $filters): self
         return $this;
     }
 
+    /**
+     * Specify which item property you want returned.
+     */
+    public function addProperty(string $property): self
```

Comment:
What about some more descriptive name? `addReturnProperty`  / `addResponseProperty` etc.?

This may appear as settings of the recommendation, not about contents of the response 🤔,